frontend/src/__tests__/actions/TablesActions.test.js

```diff
@@ -109,7 +108,7 @@ describe('TableActions general', () => {
 
     return store.dispatch(updateTableSelection(params)).then(
       expect(store.getActions()).toEqual(expect.arrayContaining(expectedActions))
-    );
+    ).catch(err => { console.log(err) });
```

Comment:
@petrican why do we need to catch & suppress?

Reply:
@teosarca Node engine 16 complains if a promise doesn't have a catch so I needed to add it and adapt the tests

Reply:
@petrican  but then... in that catch you need fail because that's a the whole point of the test:
* getActions() is successful
* getActions()  returns the expected result.

With your change, in case the getActions() fails your test is successful which is NOK.


Same things applies for the the tests below.

Reply:
You are right. I added assertion within the catch as well. This way if the test fails we will see it and with the console.log we can indicate also the origin of the failure by outputting the error that occurs in the catch.